plugins/woocommerce-blocks/assets/js/blocks/product-collection/collections/index.tsx

```diff
@@ -2,6 +2,7 @@
  * External dependencies
  */
 import { select } from '@wordpress/data';
+import { __experimentalRegisterProductCollection as registerProductCollection } from '@woocommerce/blocks-registry';
```

Comment:
As `registerProductCollection` provides default `isActive`, you can remove `isActive` function from line 36.

Reply:
Good point! Made the changes in [ec145fc](https://github.com/woocommerce/woocommerce/pull/48141/commits/ec145fc43c82c817b6b93caaff305207553ab578)